### PR TITLE
Create /home page

### PR DESCRIPTION
**Pull request by dotte-osu**
Created homepage without featured pets. (featured pets will be added in PM-43). Nothing special so far but we can change the wording and style a bit more if we want later.
image

### CODE REVIEW

File: src/routes/HomePage/HomePage.js [hunk 1]
```diff
@@ -1,11 +1,68 @@
...
+
+  const renderOverview = () => {
+    return (
+      <div className="container text-center m-3">
+        <div className="row d-flex p-2">
+
+          <div className="col-sm border rounded m-1 p-2">
+            <FontAwesomeIcon icon={faPaw} className="fa-4x text-info" />
+            <h3>Our Vision</h3>
+            <p>Intently sniff hand sleep on my human's head. Lick face hiss at owner, pee a lot, and meow repeatedly scratch at fence purrrrrr eat muffins and poutine until owner comes back chase little red dot someday it will be mine!</p>
```
Comment (xHoudek) [inline src/routes/HomePage/HomePage.js:29]: These are great haha, did you come up with these?

Did we want to update these to actual sentences? Or figure it out at a later date?

File: src/components/Header.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ export default class Header extends Component {
 
   render() {
     return (
-      <nav className='navbar navbar-expand-sm navbar-fixed-top navbar-dark bg-primary'>
+      <nav className='navbar navbar-expand-sm navbar-fixed-top navbar-dark bg-info'>
```
Comment (xHoudek) [inline src/components/Header.js:44]: I really like the new header color!

**Review by xHoudek — COMMENTED** [review]
Looks good! Just have a quick question in the comments

File: src/routes/HomePage/HomePage.js (hunk 1 above)
Comment (dotte-osu) [reply src/routes/HomePage/HomePage.js:29]: I was thinking we can figure it out later! I just googled cat ipsum haha

Comment (xHoudek) [reply src/routes/HomePage/HomePage.js:29]: Agree, I'm imaging we'll have a general "polish up" story. We can add that then

**Review by xHoudek — APPROVED** [review]

**Pull request merged**